### PR TITLE
Modifications to smoke test cases

### PR DESCRIPTION
**Pull request by vanevane**
Files deleted
-config, resourcesConfig
Files added
-resourcesEndpoints
Files modified
-resourcesLib, settings, resourcesST

### CODE REVIEW

**Comment by PedroDavid571** [comment]
I reviewed the changes and all seem to be fine.

File: test/smoke/resourcesST.js [hunk 1]
```diff
@@ -58,10 +81,11 @@ describe('Smoke Test - Resources', function(){
...
 	
-	/*Test Case
-	Title: POST resource API is present in the application
-	*/
-	it('Create a Resource', function(done){		
+	/**
+	 * Test Case
+	 * Title: POST resource API is present in the application
+	 */
+	it.only('Create a Resource', function(done){
```
Comment (JFVF) [inline test/smoke/resourcesST.js]: only should be removed

File: test/smoke/resourcesST.js [hunk 2]
```diff
@@ -69,37 +93,31 @@ describe('Smoke Test - Resources', function(){
...
-	/*Test CAse
-	Title: DELETE resource API is present in the application
-	*/
+	/**
+	 * Test Case
+	 * Title: DELETE resource API is present in the application
+	 */
 	it('Delete a Resource', function(done){
 		resources
 			.deleteResource('132465', token)
```
Comment (JFVF) [inline test/smoke/resourcesST.js]: only push complete tests

File: test/smoke/resourcesST.js [hunk 3]
```diff
@@ -37,9 +59,10 @@ describe('Smoke Test - Resources', function(){
 			});
```
Comment (JFVF) [inline test/smoke/resourcesST.js]: only submit complete tests

File: test/smoke/resourcesST.js [hunk 4]
```diff
@@ -1,16 +1,37 @@
...
-	Title: GET resources API is present in the application
-	*/
+
+	/**
+	 * Get a token
+	 *
+	 */
+	before(function(done){
+		resources
+			.getToken(auth)
```
Comment (JFVF) [inline test/smoke/resourcesST.js]: I think getToken shouldn't associated with resources... should related with another module/lib

**Comment by JFVF** [comment]
Also the comments in the functions should be following the same standards, I saw function with no description for the parameters and other with some descriptions for the parameters,

**Pull request merged**